e2e/e2e_docker_test.go

```diff
@@ -1,5 +1,3 @@
-// +build docker
```

Comment:
Better to keep this tag

Reply:
I will keep it in, but out of curiosity, what is it for?

Reply:
This tag means the test in this file will not be run when we use `go test ./...` but will be included in `go test --tags=docker ./...`. It's added to allow us to run the tests that require docker, like this, only in the environment that has docker.

We have a target `test-docker` in the top level Makefile for it.

Reply:
got it, added it back. Thanks!